packager/file/http_file.cc

```diff
@@ -297,7 +299,7 @@ void HttpFile::SetupRequest() {
   curl_easy_setopt(curl, CURLOPT_FOLLOWLOCATION, 1L);
   curl_easy_setopt(curl, CURLOPT_WRITEFUNCTION, &CurlWriteCallback);
   curl_easy_setopt(curl, CURLOPT_WRITEDATA,
-                   method_ == HttpMethod::kPut ? nullptr : &download_cache_);
+                method_ == HttpMethod::kPost ? nullptr : &download_cache_);
```

Comment:
Wouldn't this be the case for both POST and PUT?  It seems like the switch above implies that both methods would still be possible.

Or perhaps we would change this to `&download_cache_` if it's `kGet`, and null otherwise?

Reply:
The LL-DASH is going to be using PUT now, but I still switched this line to your suggestion of `&download_cache_` if it's `kGet`. Might as well clean it up to account POST and PUT while we are here :).